rclcpp/test/test_time_source.cpp

```diff
@@ -371,7 +452,6 @@ TEST_F(TestTimeSource, callback_handler_erasure) {
   EXPECT_GT(t_high.nanoseconds(), t_out.nanoseconds());
 }
```

Comment:
If you printed out the value of t_out right here, you'd see that it was usually 1s + 1000ns.  That's good enough to be between t_low and t_high, but it's not what you'd expect if everything was working correctly.  It should be 4s + 1000ns at the end of the test